### DISCUSSION TITLE
"See more from this tag" or "See less from this tag" Option

### DISCUSSION THREAD

**Discussion by Grovre** [dailydotdev/daily#865] [Feature Requests]
I like the tags I have. They all portray my interests but most of my feed is still almost all web development or clickbait hot topics after removing all related tags. I want to see more from a tag that I'm subscribed to for a reason or see less from a tag that I'm remotely interested in that is too popular and fills the feed up.

**Comment by idoshamun** [comment]
Hi @Grovre,

That's a valid concern. Thanks for sharing.
It's currently not on our roadmap directly, but we are actively working on improving our personalization of the feed. So the system should learn your preferences based on your upvotes
